### PR TITLE
Created "Get tasks by id" reference topic

### PR DESCRIPTION
**Pull request by csun-shine**

### CODE REVIEW

File: docs/api.md
Comment (rbwatson) [inline docs/api.md:1]: Added a link to the new topic.
Renamed Topic files and updated links so similar topics have similar names.

File: docs/api/tasks-get-by-task-id_csun.md [hunk 1]
```diff
@@ -0,0 +1,57 @@
+---
+layout: page
+---
+
+# Reference: Get tasks by id
+
+Returns a [`task`](tasks.md) array that contains only the task resource specified by the `id` parameter, if it exists.
```
Comment (rbwatson) [inline docs/api/tasks-get-by-task-id_csun.md:7]: I changed this from the original to describe the response value more closely.

File: docs/api/tasks-get-by-task-id_csun.md [hunk 2]
```diff
@@ -0,0 +1,57 @@
...
+| Status value | Return status | Description |
+| ------------- | ----------- | ----------- |
+| 200 | Success | Requested data returned successfully |
+| 404 | Error | Specified task record not found |
+|  ECONNREFUSED | N/A | Service is offline. Start the service and try again. |
+
+## Related Topics
+
+* [`Task`](task.md)
+* [`Get all tasks`](tasks-get-all-tasks-nikki-everett.md)
```
Comment (rbwatson) [inline docs/api/tasks-get-by-task-id_csun.md:56]: I updated the destination to match the filename of the topic.

**Review by rbwatson — APPROVED** [review]
I just made a few edits to help the page merge in to the content that's been added since you started your feature branch.

File: docs/api.md
Comment (csun-shine) [reply docs/api.md:1]: This comment is tagged as "Outdated". Does that mean that the subsequent changes to the document override these changes?

**Pull request merged**